src/compile/selection/transforms/scales.ts

```diff
@@ -34,12 +34,6 @@ const scaleBindings: TransformCompiler = {
       const extent = {selection: selCmpt.name, field: proj.field};
       scale.set('selectionExtent', extent, true);
       bound.push(proj);
-
-      // Bind both x/y for diag plot of repeated views.
-      if (model.repeater && model.repeater.row === model.repeater.column) {
```

Comment:
this seems problematic?

Reply:
This is https://github.com/vega/vega-lite/issues/6000, right?

Reply:
Perhaps, but what's the effect of removing that?

Reply:
Let's try: https://github.com/vega/vega-lite/pull/6124